pyproject.toml

```diff
@@ -19,7 +19,10 @@ classifiers = [
     "Programming Language :: Python :: 3.10",
     "Framework :: Jupyter",
 ]
-dependencies = ["jupyter_server>=1.15, <3", "jupyter_events~=0.5.0"]
+dependencies = [
+    "jupyter_server>=1.15, <3",
+    "jupyter_events"
```

Comment:
I think we can just change this to:

```
"jupyter_events>=0.5.0,<1"
```

I think we do have a minimum version requirement on `jupyter_events==0.5.0`, though I admit I cannot recall the exact reason why.

Reply:
I'm OK with dropping the `<1` version qualifier too, I just figured I'd add that in case v1 had any major breaking changes. But that could be premature reasoning.

Reply:
Yeah, given "`0.x`" releases can break APIs at any time, I'm not sure what a cap is going to do in this case, other than ensuring a release is necessary to support `jupyter_events==1.0`.  I don't think the floor is necessary at this point either, but having a floor is fine.

Reply:
I think it's more practical plausible a 1.0 release would have breaking changes than a 0.x release, but again, you're right in pointing out that it makes no technical difference.

I don't like the fact that this change leaves us open to breaking on every minor release of `jupyter_events`, but given that incompatible versions freak pip out, this is almost a case of "existing behavior".

> I don't think the floor is necessary at this point either, but having a floor is fine.

Better safe than sorry? 😁